File: ServiceBook/ServiceBook/Scripts/Service/Models/SW.js
function SW(data) {
    this.sid = null;
    this.wid = null;
    this.service = null;
    this.workingPoint = null;
    if (data != null) {
        this.sid = data.SID;
        this.wid = data.WID;
        this.service = new Service(data.Service);
        this.workingPoint=new WorkingPoint(data.WorkingPoint)
    }
}

File: ServiceBook/ClassLibrary1/Models/SW.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServiceBook.DAL.Models
{
    public class SW
    {
        [Key]
        public int SID { get; set; }
        public int WID { get; set; }
        [ForeignKey("SID")]
        public virtual Service Service { get; set; }
        [ForeignKey("WID")]
        public virtual WorkingPoint WorkingPoint { get; set; }
    }
}
